layer.md

```diff
@@ -252,32 +252,20 @@ In all other cases, the implementation MUST do the semantic equivalent of the fo
 The following is a base layer with several resources:
 
 ```text
-a/
-a/b/
-a/b/c/
-a/b/c/bar
+file1
+a/file2
+b/
 ```
 
-When the next layer is created, the original `a/b` directory is deleted and recreated with `a/b/c/foo`:
+If we then delete `file1`, `file2`, and `b/`, the next layer looks like:
 
 ```text
-a/
-a/.wh..wh..opq
-a/b/
-a/b/c/
-a/b/c/foo
+.wh.file1
+a/.wh.file2
+.wh.b
 ```
 
-When processing the second layer, `a/.wh..wh..opq` is applied first, before creating the new version of `a/b`, regardless of the ordering in which the whiteout file was encountered.
-For example, the following layer is equivalent to the layer above:
-
-```text
-a/
-a/b/
-a/b/c/
-a/b/c/foo
-a/.wh..wh..opq
-```
+Note that regardless of the path being deleted, the whiteout file is of regular filetype in the archive.
```

Comment:
Minor wording nit that makes this read more naturally:

```suggestion
Note that regardless of the path being deleted, the whiteout file is a regular file in the archive.
```

(we use the term "regular file" above on line 40, so this also updates to match that)